### DISCUSSION TITLE
ScriptReference/Application-persistentDataPath

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#100] [Notes]
# ScriptReference/Application-persistentDataPath

https://docs.unity3d.com/ScriptReference/Application-persistentDataPath.html

**Comment by docscomments** [comment]
**Comment by Shay Pierce:**  

In a UWP build running on an XboxOne, this resolves to a path like the following:  
`U:/Users/UserMgr0/AppData/Local/Packages/{APPLICATION_ID}/LocalState`

**Comment by docscomments** [comment]
**Comment by `@tenpn`:**  

When running in the editor in windows 10, this path is `~/AppData/LocalLow/{Company Name}/{Product Name}/`

**Comment by Grave18** [comment]
"Contains the path to a persistent data directory (Read-only)" - read-only is path not a folder content

**Comment by ysftulek** [comment]
in macOS player, it writes to ```com.company name.product name```, not to ```unity.company name.product name```
